client/components/site-verticals-suggestion-search/README.md

```diff
@@ -15,9 +15,9 @@ onChange( { vertical_name, vertical_slug, vertical_id } ) {
 render() {
 	return (
 		<SiteVerticalsSuggestionSearch
-			onChange={ this.onChange }
-			initialValue={ this.state.initialValue }
-			charsToTriggerSearch={ 3 }
+			autoFocus={ true }
```

Comment:
Nit: `={ true }` isn't necessary; we could just have `autoFocus` and it would do the same thing.

Reply:
Yeah, I'm being intentionally explicit here. 👍